### DISCUSSION TITLE
Any planned 3.5 version with the new cuttlefish schema additions on the horizon?

### DISCUSSION THREAD

**Discussion by SimonUnge** [rabbitmq/credentials-obfuscation#32] [General]
Hi,

Was wondering if there is any planned release of this package that will include #29 ?

**Comment by michaelklishin** [comment]
It's been a long time without a release, indeed.

**Reply by SimonUnge** [reply]
:) 

Once released I'm happy to update rabbit repo to use the new version.

**Reply by lukebakken** [reply]
Hey @SimonUnge - https://hex.pm/packages/credentials_obfuscation/3.5.0
